sdk/rust/oak/src/lib.rs

```diff
@@ -538,7 +539,11 @@ pub fn run_event_loop<T: crate::io::Decodable, N: Node<T>>(mut node: N, in_handl
 ///     # }
 /// }
 ///
-/// oak::entrypoint!(dummy => DummyNode::default());
+/// oak::entrypoint!(dummy => {
```

Comment:
This style of macro looks weird when the body involved is now just the body of the main function run by the Node.  I think this should revert to a more "normal" macro style:
```Rust
#[oak_grpc_main]
fn dummy_front() {  // no Handle available for initial node
    let dispatcher = FrontendNode::default();
    let grpc_channel = oak::grpc::server::init_default();
    oak::run_event_loop(dispatcher, grpc_channel);
}
#[oak_main]
fn dummy_inner(handle: oak::ReadHandle) {
    let dispatcher = BackendNode::default();
    oak::run_event_loop(dispatcher, handle);
}
```
(For a follow-up).

Reply:
Fixed
Translator now looks like this:
```rust
oak::entrypoint!(oak_main => |_in_channel| {
    oak::logger::init_default();
    let dispatcher = TranslatorDispatcher::new(Node);
    let grpc_channel = oak::grpc::server::init_default();
    oak::run_event_loop(dispatcher, grpc_channel);
});

oak::entrypoint!(backend_oak_main => |in_channel| {
    oak::logger::init_default();
    let dispatcher = TranslatorDispatcher::new(Node);
    oak::run_event_loop(dispatcher, in_channel);
});
```